### PR TITLE
Add color-js v1.0.1  with git auto-update config

### PR DESCRIPTION
**Pull request by x09326**
PR for #5928 
@dannyxx001 please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/brehaut/color-js
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 284
   - Count of watchers: 9
   - Count of forks: 28
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #5928, cc @brehaut

### CODE REVIEW

File: ajax/libs/color-js/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "immutable"
+  ],
+  "author": "Andrew Brehaut <andrew@brehaut.net> (https://brehaut.net/)",
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brehaut/color-js.git",
+    "basePath": "",
+    "files": [
+      "color.js"
```
Comment (pvnr0082t) [inline ajax/libs/color-js/package.json]: @x09326 I think this is not flexible. Let's use ```color*.+(js|map)```.

**Comment by x09326** [comment]
@pvnr0082t 
I changed the ```"files"```, please help me review it, thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
Thank you both.

**Pull request merged**